### DISCUSSION TITLE
Aqara Smart Door Lock D100 Zigbee - A Question

### DISCUSSION THREAD

**Discussion by LZXCorp** [Koenkk/zigbee2mqtt#22260] [Devices]
I do not own the device so I am not gonna create an issue.

https://www.aqara.com/en/product/smart-door-lock-d100-zigbee/
<img src="https://github.com/Koenkk/zigbee2mqtt/assets/107389832/19fe5b10-6944-4679-81bb-107b1218aed3" width="280">

However, I do plan on getting a smart lock in the future, so I was wondering whether anyone has this device and whether this smart lock would work with Zigbee2MQTT without the Hub, and what limitations it would have without it.

A person was asking about it here but I guess he didn't get any replies:
- https://www.reddit.com/r/Aqara/comments/18ye6ck/aqara_d100_zigbee_lock_with_z2m/
- https://www.reddit.com/r/homeassistant/comments/18ye7a8/aqara_d100_zigbee_lock_with_z2m/

I couldn't find any other discussions or questions regarding Home Assistant with the D100.

Given,
> Wireless Protocols: Zigbee, Bluetooth 5.0, NFC

suggests that only some features will only work on Zigbee, but ehh idk.

**Comment by dinhchinh82** [comment]
@LZXCorp IFAIK with some Aqara zigbee lock like A100, N100 that there is only way to bring it to the zigbee paring mode after it is connected to the Aqara hub. So I guess it's not reasonable to connect it directly to z2m without Aqara hub.

**Comment by LZXCorp** [comment]
From the following,
https://github.com/Koenkk/zigbee2mqtt/issues/13087
https://community.home-assistant.io/t/aqara-smart-door-lock-a100-zigbee/433451/2

I can conclude that none of the Zigbee Door Locks provided from Aqara will ever work with Zigbee2MQTT and ZHA.
The only solutions however based on [this](https://github.com/Koenkk/zigbee2mqtt/issues/13087#issuecomment-1217012969) is for Aqara to provide the encryption key or someone somehow reverse engineers the key to access the lock.

As it has been like 3 years already and no third-party support has been made to use the Door Locks without the Hub (HomeKit with Bluetooth works, but does not expose all of the features).

However, there is still some hope.
![image](https://github.com/Koenkk/zigbee2mqtt/assets/107389832/b29d843d-4c70-4125-ae48-c4ab8d090f00)
Of course this is still speculation and not completely confirmed, but there is at least some hope...

**Reply by zxcOvosh** [reply]
It seems that now the developers do not deny that they do not plan to add the ability to use the lock through other applications besides aqara and homekit
